Common/GPU/OpenGL/GLFrameData.h

```diff
@@ -41,6 +41,7 @@ struct GLQueueProfileContext {
 	double cpuStartTime;
 	double cpuEndTime;
 	std::string passesString;
+	int commandCounts[25];  // Can't grab count from the enum as it would mean a circular include. Might clean this up later.
```

Comment:
Could just add a static_assert at least in Common/GPU/OpenGL/GLRenderManager.cpp to catch it.  Not ideal, but avoids mucking with header issues.

-[Unknown]